addons/arsenal/functions/fnc_saveLoadout.sqf

```diff
@@ -0,0 +1,34 @@
+#include "..\script_component.hpp"
+/*
+ * Author: DartRuffian
+ * Saves a given loadout to the client's profile.
+ *
+ * Arguments:
+ * 0: Name of loadout <STRING>
+ * 1: CBA extended loadout or getUnitLoadout array <ARRAY>
+ * 2: Replace existing loadout <BOOL> (default: false)
+ *
+ * Return Value:
+ * True if loadout was saved, otherwise false <BOOL>
+ *
+ * Example:
+ * ["Current Loadout", getUnitLoadout ACE_player] call ace_arsenal_fnc_saveLoadout
+ *
+ * Public: Yes
+ */
+
+params [["_name", "", [""]], ["_loadout", [], [[]]], ["_replaceExisting", false, [false]]];
+
+if (_name == "" || {_loadout isEqualTo []}) exitWith { false };
+
+private _loadouts = profileNamespace getVariable [QGVAR(saved_loadouts), []];
```

Comment:
Need to account for when the variable doesn't exist, as atm we are modifying by reference.

Reply:
Done with just a `setVariable` at the end